src/app/sidebar/sidebar/sidebar.component.css

```
@@ -44,6 +44,34 @@ mat-slider {
   width: 100%;
 }
 
+.layer-slider {
+  margin-left: 0;
+  width: 100%;
+}
+
+.map-progress {
+  margin-top:  1rem;
+  margin-bottom: 2rem;
+  width: 100%;
+}
+
 .button-download {
   margin-bottom: 1rem;
+  margin-right: 1rem;
+}
+
+.button-cancel {
```

same rules can be summarized into one code block:

```suggestion
.button-cancel, .button-upload, .button-remove {
```